src/components/Cart.jsx

```diff
@@ -1,12 +1,50 @@
 import React from "react";
+import StripeCheckout from "react-stripe-checkout";
+import axios from "axios";
+import { Button } from "@material-ui/core";
 
 import { Order } from "./index";
 
+const STRIPE_KEY =
+  "pk_test_51JW40NGfeiZyi0bkzQs8BV34mMmKvaUDgmsf1ciXrLlMKqCs7nJncEu4H8WpCM1vnN9pxOocNckKSK04iQOChC7R00gecuqekO";
+const PAYMENT_URL = "http://localhost:5000/api/stripe/pay"; // need to set to heroku address
+const CURRENCY = "USD";
+
+// Testing purposes only need to get cart total
```

Comment:
I've worked out how to get the order total on the front end. Should we pass that figure in to the req.body as "total" so you can deconstruct it and pass in here?